java/src/jmri/jmrit/beantable/SignalMastTableAction.java

```diff
@@ -110,6 +96,15 @@ public void actionPerformed(ActionEvent e) {
         });
     }
 
+    @Deprecated
```

Comment:
Please comment on when this was deprecated in a JavaDoc comment. It'll help us know when to remove it later (sometime after 5.0 is released).